### PR TITLE
feat(quantic): origin context added to analytics of the  recommendations components

### PR DESCRIPTION
**Pull request by mmitiche**
[SFINT-5111](https://coveord.atlassian.net/browse/SFINT-5111)

The value `recommendations` has been added as the origin context of the analytics sent from the Quantic recommendations components:

oc

[SFINT-5111]: https://coveord.atlassian.net/browse/SFINT-5111?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by SimonMilord — APPROVED** [review]

File: packages/quantic/force-app/main/default/lwc/quanticRecommendationInterface/quanticRecommendationInterface.js [hunk 1]
```diff
@@ -59,6 +59,8 @@ export default class QuanticRecommendationInterface extends LightningElement {
   hasRendered = false;
   /** @type {boolean} */
   ariaLiveEventsBound = false;
+  /** @type {string} */
+  analyticsOriginContext = 'recommendations';
```
Comment (erocheleau) [inline packages/quantic/force-app/main/default/lwc/quanticRecommendationInterface/quanticRecommendationInterface.js:63]: @operecoveo @DavAtallah 
Not exactly sure how to validate this, but this is where we decide what to send to analytics under the `originContext` value

Comment (mmitiche) [reply packages/quantic/force-app/main/default/lwc/quanticRecommendationInterface/quanticRecommendationInterface.js:63]: Do we merge this @erocheleau ?

Comment (mmitiche) [inline packages/quantic/force-app/main/default/lwc/quanticRecommendationInterface/quanticRecommendationInterface.js:63]:
```suggestion
  analyticsOriginContext = 'quantic recommendations';
```

File: packages/quantic/force-app/main/default/lwc/quanticRecommendationInterface/quanticRecommendationInterface.js [hunk 2]
```diff
@@ -59,6 +59,8 @@ export default class QuanticRecommendationInterface extends LightningElement {
   hasRendered = false;
   /** @type {boolean} */
   ariaLiveEventsBound = false;
+  /** @type {string} */
+  analyticsOriginContext = 'quantic recommendations';
```
Comment (mmitiche) [inline packages/quantic/force-app/main/default/lwc/quanticRecommendationInterface/quanticRecommendationInterface.js:63]:
```suggestion
  analyticsOriginContext = 'quantic_recommendation';
```

File: packages/quantic/force-app/main/default/lwc/quanticRecommendationInterface/quanticRecommendationInterface.js (hunk 1 above)
Comment (erocheleau) [reply packages/quantic/force-app/main/default/lwc/quanticRecommendationInterface/quanticRecommendationInterface.js]: `quantic_recommendation`

**Pull request merged**